ISL.Providers.Storages.AzureBlobStorage/Services/Foundations/Files/StorageService.cs

```diff
@@ -28,7 +28,7 @@ public ValueTask CreateFileAsync(Stream input, string fileName, string container
                 await ValidateStorageArgumentsOnCreateAsync(input, fileName, container);
```

Comment:
ValidateStorageArgumentsOnCreateAsync(input, fileName, container);
```suggestion
                ValidateStorageArgumentsOnCreate(input, fileName, container);
```

Reply:
this has been updated